frontend/src/metabase/parameters/utils/parameter-values.unit.spec.js

```diff
@@ -363,6 +346,52 @@ describe("parameters/utils/parameter-values", () => {
         "parameter2 default value",
       );
     });
+
+    describe("for number filter type", () => {
+      const numberParameter = {
+        id: 111,
+        slug: "numberParameter",
+        type: "number/=",
+      };
+
+      const runGetParameterValueFromQueryParams = value =>
```

Comment:
Created this function to prevent mind-numbing repetition below.

Would be in favor of a shorter name for this function.